extension/constants.js

```diff
@@ -76,6 +76,10 @@ constants.EventType = {
   EXTENSION_BUNDLE: 'extension_install_dialog_BUNDLE_INSTALL_PROMPT',
   EXTENSION_OTHER: 'extension_install_dialog',  // Catch-all
 
+  // Defined within the extension, in background.js.
+  HTTPS: 'https',
```

Comment:
How about naming these VISITED_HTTPS and VISITED_HTTP instead? It's not immediately clear what HTTPS refers to.